includes/admin/class-wc-rest-payments-settings-controller.php

```diff
@@ -505,7 +505,7 @@ public function get_settings(): WP_REST_Response {
 				'show_woopay_incompatibility_notice'  => get_option( 'woopay_invalid_extension_found', false ),
 				'woopay_custom_message'               => $this->wcpay_gateway->get_option( 'platform_checkout_custom_message' ),
 				'woopay_store_logo'                   => $this->wcpay_gateway->get_option( 'platform_checkout_store_logo' ),
-				'woopay_enabled_locations'            => $this->wcpay_gateway->get_option( 'platform_checkout_button_locations', array_keys( $wcpay_form_fields['platform_checkout_button_locations']['options'] ) ),
+				'woopay_enabled_locations'            => $this->wcpay_gateway->get_option( 'payment_request_button_locations', array_keys( $wcpay_form_fields['payment_request_button_locations']['options'] ) ),
```

Comment:
I think the first argument had the value of `platform_checkout_button_locations` prior to the changes. Could you give some context if we should change it to `payment_request_button_locations` here? Thanks!

Reply:
Correct! Thank you for catching this. [5b5595a](https://github.com/Automattic/woocommerce-payments/pull/7565/commits/5b5595a5b48cb0738171931789cf7beba105039e)